requirements.txt

```diff
@@ -1,2 +1,5 @@
-python-language-server>=0.34.0
-mypy
+future;python_version < '3'
+flake8
```

Comment:
Where do the additional requirements come from?

Reply:
Sorry I think they came from the old pyls-mypy branch I based my changes on (see https://github.com/tomv564/pyls-mypy/pull/52). I will fix it.